### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by genadyneu**
Please review

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -46,3 +46,8 @@ Change 4 (GN)
 - - - - - - - - - - - - - - - - >
 F1 change 1
 F1 change 2
+
+
+- - - - - - - - - - - - - - - - >
+- - - - - - - - - - - - - - - - >
+F2 change 1
```
Comment (m3csl) [inline test.txt:53]: F2 ch 1 comment

File: test.txt [hunk 2]
```diff
@@ -53,3 +53,5 @@ F1 change 2
 F2 change 1
 
 F2 change 2
+
```
Comment (m3csl) [inline test.txt:56]: F2 ch3 comment 2<->3

File: test.txt (hunk 1 above)
Comment (genadyneu) [inline test.txt:53]: F2 ch1 response

File: test.txt (hunk 2 above)
Comment (genadyneu) [inline test.txt:56]: F2 ch3 comment 2<->3 response

**Pull request merged**